index.html

```diff
@@ -12,18 +12,23 @@
 <body onload="currentSlide(1)">
 <header>
     <nav>
-        <h1> Berea College Makerspace</h1>
+        
         <div class="topnav">
             <div class="nav-menu">
+                <img src="C:\Users\dixita\source\repos\bereacollegemakerspace.github.io\Makerspace\img\Duck.png"  width="100" height="100" align="middle" />

```

Comment:
You cannot use Absolute URLs in your HTML image source. Please use relative URLS.